src/webpack.d.ts

```diff
@@ -0,0 +1,738 @@
+declare module 'tapable' {
+	class Tapable {
```

Comment:
Yes. I took a look at the `webpack` and `tapable` typings in `@types` and they were terrible.